### PR TITLE
sql-statements: update `analyze` statement

### PR DESCRIPTION
**Pull request by zimulala**
### What is changed, added or deleted? (Required)
Update analyze statement and update the sqlgram of admin.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): pingcap/docs-cn#3241

### CODE REVIEW

File: sql-statements/sql-statement-analyze-table.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 
-# ANALYZE TABLE
+# ANALYZE
 
-This statement updates the statistics that TiDB builds on tables and indexes. It is recommended to run `ANALYZE TABLE` after performing a large batch update or import of records, or when you notice that query execution plans are sub-optimal.
+This statement updates the statistics that TiDB builds on tables and indexes. It is recommended to run `ANALYZE` after performing a large batch update or import of records, or when you notice that query execution plans are sub-optimal.
 
 TiDB will also automatically update its statistics over time as it discovers that they are inconsistent with its own estimates.
 
+At present, TiDB collects statistical information in two ways: full collection and incremental collection, which are realized through the `ANALYZE TABLE` and `ANALYZE INCREMENTAL TABLE` statements respectively. For detailed usage of these two statements, please refer to [introduction to statistics](/statistics.md)
```
Comment (TomShawn) [inline sql-statements/sql-statement-analyze-table.md:14]:
```suggestion
Currently, TiDB collects statistical information in two ways: full collection (implemented using the `ANALYZE TABLE` statement) and incremental collection (implemented using the `ANALYZE INCREMENTAL TABLE` statement). For detailed usage of these two statements, refer to [introduction to statistics](/statistics.md).
```

File: sql-statements/sql-statement-analyze-table.md [hunk 2]
```diff
@@ -64,6 +66,8 @@ mysql> EXPLAIN SELECT * FROM t1 WHERE c1 = 3;
 
 This statement is syntactically similar with MySQL. However, `ANALYZE TABLE` may take significantly longer to execute on TiDB, as internally it operates in a different manner.
 
+MySQL does not support the `ANALYZE INCREMENTAL TABLE` statement. TiDB supports incremental collection of statistics. For usage, please refer to [incremental collection document](/statistics.md#incremental-collection).
```
Comment (TomShawn) [inline sql-statements/sql-statement-analyze-table.md:69]:
```suggestion
MySQL does not support the `ANALYZE INCREMENTAL TABLE` statement. TiDB supports incremental collection of statistics. For detailed usage, refer to [incremental collection](/statistics.md#incremental-collection).
```

**Comment by zimulala** [comment]
PTAL @TomShawn

File: sql-statements/sql-statement-analyze-table.md [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
 
-# ANALYZE TABLE
+# ANALYZE
 
-This statement updates the statistics that TiDB builds on tables and indexes. It is recommended to run `ANALYZE TABLE` after performing a large batch update or import of records, or when you notice that query execution plans are sub-optimal.
+This statement updates the statistics that TiDB builds on tables and indexes. It is recommended to run `ANALYZE` after performing a large batch update or import of records, or when you notice that query execution plans are sub-optimal.
 
 TiDB will also automatically update its statistics over time as it discovers that they are inconsistent with its own estimates.
 
+Currently, TiDB collects statistical information in two ways: full collection and (implemented using `ANALYZE TABLE` statement) and incremental collection (implemented using the `ANALYZE INCREMENTAL TABLE` statement). For detailed usage of these two statements, please refer to [introduction to statistics](/statistics.md)
```
Comment (TomShawn) [inline sql-statements/sql-statement-analyze-table.md:14]:
```suggestion
Currently, TiDB collects statistical information in two ways: full collection (implemented using `ANALYZE TABLE` statement) and incremental collection (implemented using the `ANALYZE INCREMENTAL TABLE` statement). For detailed usage of these two statements, refer to [introduction to statistics](/statistics.md)
```

File: sql-statements/sql-statement-analyze-table.md [hunk 4]
```diff
@@ -1,16 +1,18 @@
...
 
-# ANALYZE TABLE
+# ANALYZE
 
-This statement updates the statistics that TiDB builds on tables and indexes. It is recommended to run `ANALYZE TABLE` after performing a large batch update or import of records, or when you notice that query execution plans are sub-optimal.
+This statement updates the statistics that TiDB builds on tables and indexes. It is recommended to run `ANALYZE` after performing a large batch update or import of records, or when you notice that query execution plans are sub-optimal.
 
 TiDB will also automatically update its statistics over time as it discovers that they are inconsistent with its own estimates.
 
+Currently, TiDB collects statistical information in two ways: full collection (implemented using `ANALYZE TABLE` statement) and incremental collection (implemented using the `ANALYZE INCREMENTAL TABLE` statement). For detailed usage of these two statements, refer to [introduction to statistics](/statistics.md)
```
Comment (AilinKid) [inline sql-statements/sql-statement-analyze-table.md:14]: s/using/with|by

Comment (TomShawn) [inline sql-statements/sql-statement-analyze-table.md:14]:
```suggestion
Currently, TiDB collects statistical information in two ways: full collection (implemented using the `ANALYZE TABLE` statement) and incremental collection (implemented using the `ANALYZE INCREMENTAL TABLE` statement). For detailed usage of these two statements, refer to [introduction to statistics](/statistics.md)
```

Comment (TomShawn) [reply sql-statements/sql-statement-analyze-table.md]: `using` can be more direct than `with` or `by`. For example, `The full collection is implemented using ...`.

**Pull request merged**